pkg/asset/installconfig/platformcredscheck.go

```diff
@@ -69,7 +70,7 @@ func (a *PlatformCredsCheck) Generate(dependencies asset.Parents) error {
 		if err != nil {
 			return errors.Wrap(err, "creating OpenStack session")
 		}
-	case baremetal.Name, libvirt.Name, none.Name, vsphere.Name:
+	case baremetal.Name, libvirt.Name, none.Name, vsphere.Name, alibabacloud.Name:
```

Comment:
Will the creds check be added in a follow up pr?

Reply:
I've added the creds check in the new commit.